Navigation/Profile/PhotoGalleryViewController.swift

```diff
@@ -29,38 +30,44 @@ final class PhotoGalleryViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        view.backgroundColor = .systemGray6
+        
         setUpPhotoGalleryViewController()
     }
 }
 
 extension PhotoGalleryViewController: UICollectionViewDataSource {
-    
+// MARK: Create number of sections.
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        /// allCases will return all cases that we added to the enum.
+        return PhotoSection.allCases.count
+//        return PublicationStorage.photoGalary.count
+    }
+    // MARK: Amount of items in section.
     func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        20
-//        switch PhotoSection(rawValue: section) {
-//        case .photos:
-//            return PublicationStorage.photos.count
-//        default:
-//            return 0
-//        }
+        //        guard PublicationStorage.publications.isEmpty else { return 0 }
+        switch PhotoSection(rawValue: section) {
+        case .photos:
+            return PublicationStorage.photoGalary.count
+        default:
+            return 0
+        }
     }
-    /// Creating a cell.
+// MARK: Creating a cell.
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
         /// Метод dequeue, чтобы не переполнялась память.
         let cell: PhotoCollectionViewCell = collectionView.dequeueReusableCell(
             withReuseIdentifier: String(describing: PhotoCollectionViewCell.self),
             for: indexPath
         ) as! PhotoCollectionViewCell
         
-//        cell.photo = PublicationStorage.photos[indexPath.section][indexPath.item]
-        
-//        cell.backgroundColor = .red
+        cell.photoGalary = PublicationStorage.photoGalary[indexPath.section][indexPath.item]
```

Comment:
after you fix photoGallery in PublicationStorage as I requested, and fix the cell it would be 
cell.photo = PublicationStorage.photoGallery.photos[indexPath.item]